js/plugins/fixed_toolbar/plugin.js

```diff
@@ -0,0 +1,16 @@
+CKEDITOR.plugins.add('fixed_toolbar', {
+  init: (editor) => {
+    CKEDITOR.on('instanceReady', (e) => {
+      if (null === e.editor.container.find('.cke_top').getItem(0)) {
+        return;
+      }
+      const toolBar = e.editor.container.find('.cke_top').getItem(0).$;
+      toolBar.style.position = 'sticky';
+
+      const dialog = jQuery('.MuiDialogTitle-root');
```

Comment:
This assumes jQuery as a dependency. Do we have that loaded somewhere?

Reply:
https://git.drupalcode.org/project/drupal/-/blob/8.8.x/core/modules/ckeditor/ckeditor.libraries.yml#L10

Reply:
Danke.